src/app/shared/components/custom-table/custom-table.component.html

```diff
@@ -8,29 +8,49 @@
       [page]="1"
       style="width: 100%; height: 50rem"></ng2-pdfjs-viewer>
   </div>
-  <table mat-table [dataSource]="dataSource" matSort class="mat-elevation-z8">
+  <table
+    mat-table
+    [dataSource]="dataSource"
+    matSort
+    (matSortChange)="announceSortChange($event)"
+    [matSortActive]="columnSchema[0].key"
+    matSortDirection="desc"
+    class="mat-elevation-z8">
     <ng-container
       *ngFor="let column of columnSchema"
       [matColumnDef]="column.key">
       <th
         mat-header-cell
         *matHeaderCellDef
         mat-sort-header
-        sortActionDescription="{{ 'Sort by ' + column.label }}">
```

Comment:
directives then classes then attributes followed by data binding and events at last. follow this sequence